data/json/effects.json

```diff
@@ -139,7 +139,8 @@
         "type": "effect_type",
         "id": "shrieking",
         "name": ["Shrieking"],
-        "desc": ["AI tag used for screecher sounds.  This is a bug if you have it."]
+        "desc": ["AI tag used for screecher sounds.  This is a bug if you have it."],
+        "show_in_info": true
```

Comment:
The name `"show_in_info"` has a pretty wide scope (so do the effects), but it's used only for monsters in their descriptions.